juju/utils.py

```diff
@@ -93,20 +95,31 @@ class IdQueue:
     ID.
     """
 
-    def __init__(self, maxsize=0):
-        self._queues = defaultdict(partial(jasyncio.Queue, maxsize))
-
-    async def get(self, id_):
+    _queues: dict[int, asyncio.Queue[dict[str, Any] | Exception]]
+
+    def __init__(self):
+        self._queues = defaultdict(asyncio.Queue)
```

Comment:
Behaviour change? Or is it always called with the default maxsize internally, and we're confident it isn't used externally?

Reply:
yes maxsize is always the default.